src/sentry/integrations/base.py

```diff
@@ -87,14 +79,14 @@ def feature_flag_name(f: str | None) -> str | None:
             return f"integrations-{f}"
         return None
 
-    def _asdict(self) -> dict[str, Sequence[Any]]:
-        metadata = super()._asdict()
+    def asdict(self) -> dict[str, Sequence[Any]]:
```

Comment:
I think this is probably `dict[str, Any]` -- or could be a `TypedDict` if you want to write one out for the class (though maybe overkill for now?)

Reply:
`dict[str, Any]` it is